.github/workflows/codegen.yml

```
@@ -99,26 +99,16 @@ jobs:
         run: |
           echo "${LATEST_AWS_SDK_GO_TAG}" > .latest-tag-aws-sdk-go
 
-      - name: Open Pull Request
-        uses: peter-evans/create-pull-request@v3
+      - name: Commit files
+        run: |
+          git config --local user.email "noreply@github.com"
+          git config --local user.name "github-actions[bot]"
+          git add lib/aws/generated/*
+          git add .latest-tag-aws-sdk-go
+          git commit -m "Update services based on ${{ env.LATEST_AWS_SDK_GO_TAG }} of AWS Go SDK"
```

```suggestion
          echo "Update services based on ${{ env.LATEST_AWS_SDK_GO_TAG }} of AWS Go SDK" >> commit-msg
          echo >> commit-msg
          echo "Reference: https://github.com/aws/aws-sdk-go/releases/tag/${{ env.LATEST_AWS_SDK_GO_TAG }}" >> commit-msg
          git commit -F commit-msg
```

This is slightly more complicated but does the trick.

---

Looks good! I'll apply the same change to aws-erlang later 👍 Thanks for the fast feedback 🙏